test/swath-projector/version.txt

```diff
@@ -1 +1 @@
-0.1.10
+0.2.0
```

Comment:
Thoughts here. Maybe a major version bump is more appropriate? (I guess the reference files aren't backwards compatible)

Reply:
I bumped this to 1.0.0. I talked myself into this being a big-ish deal. Even though I totally agree that the semantic version numbers here aren't incredibly thorough.